.github/workflows/k6-perf-test-cloud-assets.yaml

```diff
@@ -0,0 +1,55 @@
+name: K6 Performance Cloud Test Assets
+on:
+  workflow_dispatch:
+    inputs:
+      requests-count:
+        description: 'Number of HTTP requests to send. Since the number of iterations is indirectly controlled by the duration and rate, it is possible to be slightly inaccurate'
+        type: number
+        required: true
+        default: 10
+      assets-no-metadata-count:
+        description: 'Number of assets ids without metadata to include in each request.'
+        type: number
+        required: true
+        default: 6
+      assets-on-chain-metadata-count:
+        description: 'Number of assets ids with on-chain metadata to include in each request.'
+        type: number
+        required: true
+        default: 3
+      assets-off-chain-metadata-count:
+        description: 'Number of assets ids with off-chain metadata to include in each request.'
+        type: number
+        required: true
+        default: 1
+        
+env:
+  # Only running on dev-mainnet for now
+  TARGET_ENV: 'dev'
+  TARGET_NET: 'mainnet'
+  REQUESTS_COUNT: ${{ inputs.requests-count || 10 }}
```

Comment:
Which is the purpose of `|| 10` when it is specified as default value for given input?

Reply:
In this case, where the only way to run the test is via the "Run action" button, there isn't much use for it.
But when we'll add a scheduled run (cron), the inputs and their defaults will be ignored, so this will ensure it's initialized.

Reply:
Nice to know: thank you!

I thought when run on schedule, default values were taken.